ide/plugins/java-format/src/content/formats/webdriver-testng.js

```diff
@@ -0,0 +1,523 @@
+/*
+ * Formatter for Selenium 2 / WebDriver Java client.
+ */
+
+if (!this.formatterType) {  // this.formatterType is defined for the new Formatter system
+  // This method (the if block) of loading the formatter type is deprecated.
+  // For new formatters, simply specify the type in the addPluginProvidedFormatter() and omit this
+  // if block in your formatter.
+  var subScriptLoader = Components.classes["@mozilla.org/moz/jssubscript-loader;1"].getService(Components.interfaces.mozIJSSubScriptLoader);
+  subScriptLoader.loadSubScript('chrome://selenium-ide/content/formats/webdriver.js', this);
+}
+
+function useSeparateEqualsForArray() {
+  return true;
+}
+
+function testClassName(testName) {
+  return testName.split(/[^0-9A-Za-z]+/).map(
+      function(x) {
+        return capitalize(x);
+      }).join('');
+}
+
+function testMethodName(testName) {
+  return "test" + testClassName(testName);
+}
+
+function nonBreakingSpace() {
+  return "\"\\u00a0\"";
+}
+
+function array(value) {
+  var str = 'new String[] {';
+  for (var i = 0; i < value.length; i++) {
+    str += string(value[i]);
+    if (i < value.length - 1) str += ", ";
+  }
+  str += '}';
+  return str;
+}
+
+Equals.prototype.toString = function() {
+  if (this.e1.toString().match(/^\d+$/)) {
+    // int
+    return this.e1.toString() + " == " + this.e2.toString();
+  } else {
+    // string
+    return this.e1.toString() + ".equals(" + this.e2.toString() + ")";
+  }
+};
+
+Equals.prototype.assert = function() {
+  return "assertEquals(" + this.e1.toString() + ", " + this.e2.toString() + ");";
+};
+
+Equals.prototype.verify = function() {
+  return verify(this.assert());
+};
+
+NotEquals.prototype.toString = function() {
+  return "!" + this.e1.toString() + ".equals(" + this.e2.toString() + ")";
+};
+
+NotEquals.prototype.assert = function() {
+  return "assertThat(" + this.e1.toString() + ", is(not(" + this.e2.toString() + ")));";
+};
+
+NotEquals.prototype.verify = function() {
+  return verify(this.assert());
+};
+
+function joinExpression(expression) {
+  return "join(" + expression.toString() + ", ',')";
+}
+
+function statement(expression) {
+  var s = expression.toString();
+  if (s.length == 0) {
+    return null;
+  }
+  return s + ';';
+}
+
+function assignToVariable(type, variable, expression) {
+  return type + " " + variable + " = " + expression.toString();
+}
+
+function ifCondition(expression, callback) {
```

Comment:
Not possible. This is a plugin for Webdriver TestNG formatter. you should implement these methods. Similarly java-backend-junit4.js, java-rc-junit4.js, java-rc.js has implemented these methods.